app/src/components/Table/Table.jsx

```diff
@@ -23,17 +69,26 @@ function Table({ rows, columns, highlightedIndex, onRowClicked, clickable }) {
       <thead>
         <tr>
           {columns.map(({ key, label, className }) => {
+            const customClass = (className && className()) || '';
+            const arrowClass = `arrow-${sortData.value}`;
             return (
-              <th className={className && className()} key={`col-${key}`}>
+              <th className={customClass} key={`col-${key}`} onClick={() => handleClick(key)}>
                 {label || label === '' ? label : capitalize(key)}
+                <div className={`${customClass} arrow`}>
```

Comment:
Following my comment below about modifiers, you could then use the `classNames` package imported at the top of the file to do something like:

`
className = {"arrow", {"-ascending": isAscending, "-descending": isDescending}`